docs/root/intro/arch_overview/load_balancing.rst

```diff
@@ -154,29 +154,19 @@ priority may also be specified. When endpoints at the highest priority level (P=
 traffic will land on endpoints in that priority level. As endpoints for the highest priority level
 become unhealthy, traffic will begin to trickle to lower priority levels.
 
-Currently, it is assumed that each priority level is over-provisioned by the
-:ref:`overprovisioning factor <arch_overview_load_balancing_overprovisioning_factor>`.
-With default factor value 1.4, if 80% of the endpoints are healthy, the priority level is still considered
-healthy because 80*1.4 > 100. As the number of healthy endpoints dips below 72%, the health of the priority level
-goes below 100. At that point the percent of traffic equivalent to the health of P=0 will go to P=0
-and remaining traffic will flow to P=1.
-
-It is important to understand how Envoy evaluates priority levels' health. Each priority level is assigned
-a health value which basically is a percentage of healthy hosts in relation to total number of hosts in a given 
-priority level multiplied by overprovisioning factor of 1.4. 
-A priority level's health is integer value between 0% and 100%. When there are more than one priority levels
-in a cluster, Envoy adds all priority levels' health values and caps it at 100%. This is called normalized total health. Value 0% of 
-normalized total health means that all hosts in all priority levels are unhealthy. Value 100% of normalized total health may 
-describe many situations: all levels have health of 100% or 4 levels have health value of 30% each.
-When normalized total health value is 100%, Envoy assumes that there are enough healthy hosts in all priority 
-levels to handle the load. Not all hosts need to be in one priority as Envoy distributes traffic across priority 
-levels based on each priority level's health value.
-In order for load distribution algorithm and normalized total health calculation to work properly, the number of hosts
-in each priority level should be close. Envoy assumes that for example 100% healthy priority level P=1 is able to take
-the entire traffic from P=0 should all its hosts become unhealthy. If P=0 has 10 hosts and P=1 has only 2 hosts, P=1 may be unable
-to take the entire load from P=0, even though P=1 health is 100%.
-
-Assume a simple set-up with 2 priority levels, P=1 100% healthy.
+The system can be overprovisioned with a configurable
+:ref:`overprovisioning factor <arch_overview_load_balancing_overprovisioning_factor>`, which
+currently defaults to 1.4 (this document will assume this value). If 80% of the endpoints in a
+priority level are healthy, that level is still considered fully healthy because 80*1.4 > 100.
+So, level 0 endpoints will continue to receive all traffic until less than ~71.4% of them are
+healthy.
+
+The priority level logic works with integer 'health' scores. The health score of a level is
+(level's percent of healthy endpoints) * (overprovisioning factor), capped at 100. When the system
+is relatively healthy, P=0 endpoints receive (level 0's health score) percent of the traffic, with
```

Comment:
internally this is referred to as "priority load": https://github.com/envoyproxy/envoy/blob/master/include/envoy/upstream/load_balancer.h#L65

maybe it makes sense to talk about it in terms of the priority load being the % sent to each dc and it's computed based on health % + overprovisioning factor?

Reply:
They're used somewhat interchangeably